packages/playground/src/types/index.ts

```diff
@@ -71,6 +72,8 @@ export interface CaproverWorker {
   name: string;
   solution?: solutionFlavor;
   dedicated?: boolean;
+  rentedBy?: number;
+  rentedByMe?: boolean;
```

Comment:
do we still need rentedByMe?

Reply:
it's for interface as it used for the switch